.github/workflows/check.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2021-09-01
```

Comment:
not sure, i would prefer to retain specific nightly, even if it is old. either we risk to break ci at some point and also ci may be slower checking out new rust each day